field/src/extension/quadratic.rs

```diff
@@ -4,6 +4,7 @@ use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Sub, SubAssi
 
 use num::bigint::BigUint;
 use num::Integer;
+#[cfg(feature = "rand")]
 use rand::Rng;
```

Comment:
Just a nit, but what do you think of having this import in the method(s) that use it? Either generally or just in files like this with a single rand method. It's a bit unusual, but might be nice to reduce the cfg guards.

Reply:
The method signature (takes generic type parameter `R: Rng`) depends on the trait being in-scope. I can just move this import to the bottom of the imports so it's separated and looks cleaner if that would be preferrable.